.github/workflows/ci-pr.yml

```diff
@@ -0,0 +1,113 @@
+name: CI Pull Request
+on: pull_request
```

Comment:
Should this run for all pr events? Or just some? ie create and edit vs review added etc

Reply:
ah meant to have it run on open pr (create) as well as code push. Didn't think adding review would trigger this too. Will be more specific.

Reply:
Yeah I was surprised at how many events they do offer for PRs. Definitely useful for other stuff like adding/removing labels etc

Reply:
Are we leaving this to trigger on all events for now?

Reply:
By default, it's triggered on `created`, `synchronized` and `reopened` events and that's what we want, right?

Reply:
I guess I could make it more explicit by spelling out all those events..

Reply:
ah I see yeah I forgot those are the exact events here. Disregard!